Makefile

```diff
@@ -11,57 +11,6 @@ linux-static:
 check:
 	cargo rustc -- -D warnings
 
-## RELEASE step 1: Start our workflow generating artifacts.
```

Comment:
We don't need this for release, but it could be handy to be able to easily build artifacts outside of the release process. Maybe we should keep the targets for steps 1-3 ? Or documenting it in the README would be sufficient? WDYT?

Reply:
You can still generate artifacts by manually triggering via the UI: https://github.com/asfaload/asfald/actions/workflows/ci.yml

Keeping a duplicate behavior might not be ideal without forgetting to keep it in sync.

Reply:
You're right, thx